packages/zoe/src/zoeService/utils.d.ts

```diff
@@ -13,22 +14,24 @@ type ContractFacet<T extends {} = {}> = {
 /**
  * Installation of a contract, typed by its start function.
  */
-declare const StartFunction: unique symbol;
-export type Installation<SF> = {
-  getBundle: () => SourceBundle;
-  getBundleLabel: () => string;
-  // because TS is structural, without this the generic is ignored
-  [StartFunction]: SF;
-};
-export type Instance<SF> = Handle<'Instance'> & {
-  // because TS is structural, without this the generic is ignored
-  [StartFunction]: SF;
-};
+export type Installation<SF extends ContractStartFunction | unknown> = Tagged<
```

Comment:
I'm not sure how to read this. `X | unknown` is `unknown` for any `X`, isn't it?

Reply:
this used to be `SF` which was `any`. I constrained it to `SF extends ContractStartFunction` but some places explicitly pass `unknown`. So this continues to allow that.